_meta/beat.reference.yml

```diff
@@ -24,6 +24,9 @@ apm-server:
   # Maximum number of requests permitted to be sent to the server concurrently.
   #concurrent_requests: 5
 
+  # Maximum number of new connections to accept simultaneously
```

Comment:
Could you add a note that `0` means unlimited (even though it is kind of expected).

Reply:
done